mlflow/server/js/src/experiment-tracking/components/HomeView.js

```diff
@@ -50,23 +70,25 @@ class HomeView extends Component {
     return (
       <div className='outer-container' style={{ height: containerHeight }}>
         <div>
-          {this.state.listExperimentsExpanded ? (
-            <ExperimentListView
-              activeExperimentId={this.props.experimentId}
-              onClickListExperiments={this.onClickListExperiments}
-            />
-          ) : (
-            <i
-              onClick={this.onClickListExperiments}
-              title='Show experiment list'
-              style={styles.showExperimentListExpander}
-              className='expander fa fa-chevron-right login-icon'
-            />
-          )}
+          <PageContainer>
+            {this.state.listExperimentsExpanded ? (
+              <ExperimentListView
+                activeExperimentId={this.props.experimentIds && this.props.experimentIds[0]}
+                onClickListExperiments={this.onClickListExperiments}
+              />
+            ) : (
+              <i
+                onClick={this.onClickListExperiments}
+                title='Show experiment list'
+                style={styles.showExperimentListExpander}
+                className='expander fa fa-chevron-right login-icon'
+              />
+            )}
+          </PageContainer>
```

Comment:
@harupy good catch; sorry I missed this on the conversion! It might be cleaner to just do this:

```suggestion
          <Spacer />
          {this.state.listExperimentsExpanded ? (
            <ExperimentListView
              activeExperimentId={this.props.experimentIds && this.props.experimentIds[0]}
              onClickListExperiments={this.onClickListExperiments}
            />
          ) : (
            <i
              onClick={this.onClickListExperiments}
              title='Show experiment list'
              style={styles.showExperimentListExpander}
              className='expander fa fa-chevron-right login-icon'
            />
          )}
```

Reply:
@xanderwebs Thanks for the suggestion!

Reply:
Applied the suggestion:

<img width="807" alt="image" src="https://user-images.githubusercontent.com/17039389/161649979-a8316d99-a46d-4b71-838d-4afb51a33e88.png">

Looks good!